tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,157 @@
+                    </span>
+                  ) : <></>}
+                </>
+              );
+              case 'released': return (
+                <>
+                  {data.doi
+                    ? (
+                      <span className="badge badge-info mr-2">
+                        <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                      </span>
+                    ) : <></>
+                    }
+
+                  <span className="text-muted" title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                    {`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                  </span>
+                </>
+              );
+              case 'retracted': return (
+                <>
+                  {data.doi ? (
+                    <span className="badge badge-info mr-2">
+                      <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                    </span>
+                  ) : <></>}
+                  <div>
+                    <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                      {`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                    </span>
+                  </div>
+                  <div>
+                    <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Retracted on ${moment(data.retracted_date, 'YYYY-MM-DD').format('ll')}`}>
+                      {`Retracted on ${moment(data.retracted_date, 'YYYY-MM-DD').format('ll')}`}
+                    </span>
+                  </div>
+
+                </>
+              );
+              default: return (
+                <>
+                  {data.doi ? (
+                    <span className="badge badge-info mr-2">
+                      <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                    </span>
+                  ) : <></>}
+
+                  <span className="text-muted" title={`Scheduled Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                    {`Scheduled Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                  </span>
+                </>
+              );
+            }
+          })()}
+        </div>
+        <div className="card-footer">
+          {(() => {
+            switch (publicationType) {
+              case 'draft': return (
+                <Fragment>
+                  <button type="button" onClick={e => handleResume(e, data.id)} className="btn btn-primary btn-sm mr-2 mb-1">
+                    <i className="fa fa-edit mr-1" />
+                    Resume Draft
+                  </button>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 40 spaces but found 22.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                                        </button>
```